crates/xmtp-networking/src/lib.rs

```diff
@@ -70,16 +71,20 @@ mod tests {
                 .unwrap();
 
             // Sleep to give the response time to come back
-            std::thread::sleep(std::time::Duration::from_millis(100));
+            tokio::time::sleep(std::time::Duration::from_millis(100)).await;
```

Comment:
How did this differ from the std::thread::sleep?

Reply:
I realized in testing that this was also blocking the reading of the stream (everything goes to sleep), whereas this will yield and let other async processes continue working.

Reply:
Gotcha!